### DISCUSSION TITLE
Scroll long file names in code viewer

### DISCUSSION THREAD

**Discussion by eygraber** [community/community#60559] [Mobile]
If a filename is long there's no way to see it fully. Being able to scroll it out pinch to zoom to see it would be very helpful.

**Comment by Akash1134** [comment]
Hi @eygraber ,

This is a similar feedback you've already shared in this discussion: [Allow pinch to zoom for font size when viewing a file](https://github.com/orgs/community/discussions/60558)

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
